### PR TITLE
Grammar fixes in README.md + Reduce imports + 14 lines save (Minor fixes)

### PR DESCRIPTION
**Pull request by donno2048**
## Changelog / Overview

Unnecessary since the change is tiny.

## Motivation

Fixes in imports and line reduces make it easier to understand what are the "dependencies" of each functionality in the manim package and saves space.

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,166 +1,166 @@
...
+## Help with Manim
+
+If you need help installing or using Manim, feel free to reach out to our [Discord
+Server](https://discord.gg/mMRrZQW) or [Reddit Community](https://www.reddit.com/r/manim). If you would like to submit a bug report or feature request, please open an issue.
+
+## Contributing
+
+Contributions to Manim are always welcome. In particular, there is a dire need for tests and documentation. For contribution guidelines, please see the [documentation](https://docs.manim.community/en/stable/contributing.html).
+
+Most developers on the project use [Poetry](https://python-poetry.org/docs/) for management. You'll want to have poetry installed and available in your environment. You can learn more about `poetry` and how to use it at its [documentation](https://docs.manim.community/en/stable/installation/for_dev.html).
```
Comment (donno2048) [inline README.md:158]: All the changes to the README are in between those lines, I guess the problem was that my machine accidentally replaced all the line-endings

**Comment by behackl** [comment]
Hello! Thank you for your contribution!

In order to get this reviewed and merged eventually, please make sure to push a version without modified line endings in different files. You might have to change your editor from CRLF to just LF (or alternatively, there should also be a corresponding setting with `git` itself).

**Review by jsonvillanueva — CHANGES_REQUESTED** [review]
Hey thanks for the contribution!

Please only fix one issue per PR so we can review this more quickly. In particular, you should keep this PR specific to Grammar fixes in README.md and open new PRs for Reducing imports/line saves (though I'll let you know now that those line saves are actually necessary and will break Manim if done)

File: .gitattributes [hunk 2]
```diff
@@ -1 +1,2 @@
 manim/grpc/gen/** linguist-generated=true
+* eol=crlf
```
[the remark this answers is not in the document]
Comment (donno2048) [reply .gitattributes]: I **did** use lf but this caused some images to get corrupted so I changed it to crlf just to fix it

**Comment by donno2048** [comment]
> Please only fix one issue per PR

Sure

**Pull request closed without merging**